frontend/src/methods/features.ts

```diff
@@ -31,6 +33,33 @@ export const setupWorkloadProxyingEnabledFeatureWatch = (): Ref<boolean> => {
   })
 }
 
+let userPilotInitialized = false
+
+export const initializeUserPilot = async () => {
+  if (!currentUser.value) {
+    return
+  }
+
+  if (!userPilotInitialized) {
+    const featuresConfig = await ResourceService.Get<Resource<FeaturesConfigSpec>>({
```

Comment:
is it possible since this is async that refreshUser is called twice and we initialize pilot twice or so?